### PR TITLE
Make the island ranks configurable

### PR DESCRIPTION
**Pull request by MertUnverdi**

### CODE REVIEW

**Review by PeachesMLG — APPROVED** [review]

File: src/main/java/com/iridium/iridiumskyblock/configs/inventories/IslandRanksInventoryConfig.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     */
+    public Background background;
+    /**
+     * The item for Owner Rank
+     */
+    public Item owner;
+    /**
+     * The item for Co-Owner Rank
+     */
+    public Item co_owner;
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/configs/inventories/IslandRanksInventoryConfig.java:31]: Should be coOwner. Java variables are always lowerCamelCase

**Review by dlsf — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/configs/inventories/IslandRanksInventoryConfig.java (hunk 1 above)
Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/configs/inventories/IslandRanksInventoryConfig.java:31]: ya right

**Comment by PeachesMLG** [comment]
You need to change the references aswell :P

File: src/main/java/com/iridium/iridiumskyblock/gui/IslandRanksGUI.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    @Override
+    public void addContent(Inventory inventory) {
+        inventory.clear();
+        IslandRanksInventoryConfig islandRanks = IridiumSkyblock.getInstance().getInventories().islandRanksGUI;
+        InventoryUtils.fillInventory(inventory, islandRanks.background);
+        List<User> members = IridiumSkyblock.getInstance().getIslandManager().getIslandMembers(island);
+        inventory.setItem(islandRanks.owner.slot, ItemStackUtils.makeItem(islandRanks.owner,
+                Collections.singletonList(new Placeholder("members", island.getOwner().getName()))));
+        inventory.setItem(islandRanks.co_owner.slot, ItemStackUtils.makeItem(islandRanks.co_owner,
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/gui/IslandRanksGUI.java:60]: You never changed the names here

**Review by dlsf — APPROVED** [review]

**Pull request merged**